docs/tutorials/upgrading-ev3dev-from-command-line.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Upgrading Ev3dev from the Command Line
+subject: System
+---
+
+{% include icon.html type="warning" %}
+By upgrading ev3dev you are also upgrading the drivers, so there may be changes that break user software. 
```

Comment:
You are not necessarily upgrading drivers and drivers are not the only thing that can break other software.

Furthermore, my advice is the Linux Mint philosophy: if everything is working and you are happy, then **don't upgrade**. Only upgrade if A) something is not working for you and you think that upgrading a package will fix the problem or B) you want to help with testing ev3dev and you plan on reporting bugs when you find something that is broken.